.github/vmctl

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
You can remove this entire file. The new workflow downloads the latest version automatically

Reply:
stand by